src/components/tree-item/tree-item.scss

```diff
@@ -155,6 +149,19 @@
   }
 }
 
+// selection-mode none, dropdown without children
```

Comment:
IMO, the selector is readable, so we can drop this comment. Also, there's no dropdown here, right?